src/parsers/parser_nodes.ts

```diff
@@ -26,25 +27,30 @@ export class ParsedRange {
 
 // export type ParsedNodeType = 'expect' | 'describe' | 'it' | 'ROOT';
 
-export const ParsedNodeTypes = {
+export const ParsedNodeTypes: {
+  describe: 'describe';
+  expect: 'expect';
+  it: 'it';
+  root: 'root';
+} = {
   describe: 'describe',
   expect: 'expect',
   it: 'it',
   root: 'root',
 };
```

Comment:
It has to be tested, but this could be replaced by either an enum or `export type ParsedNodeTypes = 'describe' | 'expect' | ...`.

Reply:
Agreed.  I was tempted to rewrite some of the parser code relying on TS more, but I see that #30 is already rewriting the parser code.  It would probably make more sense to merge those changes first.

Reply:
Actually having read that PR, I'm inclined to wait until it is merged because it will remove the typescript parser. That reduces the delta in this PR.